sql/catalyst/pom.xml

```diff
@@ -103,6 +103,12 @@
       <groupId>commons-codec</groupId>
       <artifactId>commons-codec</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.univocity</groupId>
+      <artifactId>univocity-parsers</artifactId>
+      <version>2.7.3</version>
+      <type>jar</type>
+    </dependency>
```

Comment:
I added the dependency only because I have to move `UnivocityParser` from `sql/core` to `sql/catalyst` because it is not accessible from `sql/core`. Please, tell me what is right approach here?

Reply:
Ideally we should just make CSV one as a separate external module and this should be the right way given the discussion.

The current change wouldn't necessarily is blocked but I can see the point of moving the dependency makes further refactoring potentially harder as pointed out. Looks many people agreed upon separating them.

The concern here is,  it sounds we are stepping back from the ideal approach.